thijsvanloef/palworld-server-docker
# Typos in readme, K8s, & docker compose

## Context 

* Max player count listed in documentation is 31 instead of 32.
* Server name can be set for non-community server.
* Fixed spacing typos in readme
* K8s was outdated

## Choices

* Had to update documentation
* Updated K8s to match docker-compose

## Test instructions

1. Verify documentation
2. Run container
3. Join game and verify server name is set within server for non-community server by hitting escape
![20240125201709_1](https://github.com/thijsvanloef/palworld-server-docker/assets/18587674/42c316e6-0989-411e-b877-e3228a75dd07)



## Checklist before requesting a review
